### DISCUSSION TITLE
Could anyone tell me how to add translations for kyrgyz language?

### DISCUSSION THREAD

**Discussion by Soyuzbek** [encode/django-rest-framework#8345] [Question & Answer]
Hello Django Rest Framework community!

I just finished translating strings on transifex to kyrgyz language. And I would like to learn how to include kyrgyz language to DRF itself.
Can anyone explain how it works to add new language to DRF? Does transifex add new language to DRF automatically or I have to fork DRF and compile django.po files?

**Comment by lovelydinosaur** [comment]
Ah great - thanks - you don't need to do anything else.
We'll need a new release before they'll be included tho.
